### ISSUE TITLE
NVDA cannot report comments With UIA enabled in Microsoft word

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#11789]
### Steps to reproduce:

With UIA enabled, Enter some text in Microsoft word and select a part, then insert a comment.
Press nvda+ space and Move the leftarrow and right arrow. nvda report has comments, Press nvda+alt+C at the comment.

### Actual behavior:
NVDA does not Report comments.

### Expected behavior:
NVDA should report comments.

#### NVDA installed/portable/running from source:
Installed and portable

#### NVDA version:
2020.3

#### Windows version:
Windows 10 20H2 (64bit) build 19042.572

#### Name and version of other software in use when reproducing the issue:
Microsoft 365 MSO (16.0.13231.20372) 64 Bit

#### Other information about your system:
None.

#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I get the same behavior with NVDA 2020.2, 2019.3.1 and 2019.2.1
#### If addons are disabled, is your problem still occuring?
Yes.
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes.

**Comment by Andre9642** [comment]
Duplicate of #9285

**Comment by zstanecic** [comment]
Hi,

As I recall, there is a pull request for this, but no review or activity was recently there.

**Comment by cary-rowen** [comment]
oh, If there is already a solution to close this issue.

**Comment by Adriani90** [comment]
Closing as der uplicate
